test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkOperationController.cs

```diff
@@ -520,4 +521,33 @@ public ITestActionResult Post([FromBody] Student student)
             return Created(student);
         }
     }
-}
\ No newline at end of file
+
+    public class CustomersController : TestODataController
+    {
+        [ODataRoute("Customers({key})")]
+        [HttpPatch]
+        [EnableQuery]
+        public ITestActionResult Patch(int key, Delta<Customer> delta)
+        {
+            Assert.True(delta.TryGetPropertyValue("Orders", out var orders));
+
```

Comment:
You could add an Assert here to verify the expected number of items in the collection, otherwise `changes[0]`, `changes[1]`, and `changes[2]` could easily throw index out of range exception, which is not how we'd want the test to fail

Reply:
Let me add such checks in the version update PR.